hugo/content/guides/multiple-languages.md

```diff
@@ -0,0 +1,454 @@
+---
+title: "Multiple dependent languages"
+weight: 400
+---
+
+This tutorial is about integrating multiple dependent languages in one Langium project.
+
+One common situation were it makes sense to create dependent languages is when you only want to read concepts in one language and predefine them in another file (probably also a built-in one). Think of splitting SQL into a defining `CREATE TABLE table (...)`) and a reading part (`SELECT * FROM table`).
+
+> Note, that for n independent languages can be simply made by creating n independent Langium projects.
+
+## Our plan
+
+The entire change touches several files. Let's summarize what needs to be done:
+
+1. the **grammar** (the `*.langium` file) needs to be split into the three parts that were discussed above
+2. the **Langium configuration** (the `langium-config.json` file in the Langium project root) needs to split the language configuration into thre new
+3. the **`Djinject` module** of your language (`XXX-module.ts`) needs to create the new language services as well.
```

Comment:
Are we already using Djinject in the current Langium version?

Reply:
I will call it `module file`